tests/test_cases/simulation/test_mpi_ensemble.cu

```diff
@@ -0,0 +1,289 @@
+#include <chrono>
+#include <thread>
+
+#ifdef FLAMEGPU_ENABLE_MPI
+#include <mpi.h>
+#endif
+
+#include "flamegpu/flamegpu.h"
+#include "gtest/gtest.h"
+
+
+namespace flamegpu {
+namespace tests {
+namespace test_mpi_ensemble {
+/**
```

Comment:
Do we want to split this out to a separate test binary so we can run it with more than one rank, without running every other test N times as well? 

We could configure CTest to also use mpirun -n 4 and mpirun n 1 versions separately.

Reply:
alternatively we could make it a standalone binary which just returns an appropraite error code for pass / fail, and orchestrate that via CTest if Googletest itself is causing issues.

Reply:
to be discussed with Paul I guess

Reply:
Test suite has been split out.